src/mbgl/util/tile_cover.hpp

```diff
@@ -18,5 +18,8 @@ int32_t coveringZoomLevel(double z, SourceType type, uint16_t tileSize);
 std::vector<UnwrappedTileID> tileCover(const TransformState&, int32_t z);
 std::vector<UnwrappedTileID> tileCover(const LatLngBounds&, int32_t z);
 
+// Compute only the count of tiles needed for tileCover
+unsigned long tileCount(const LatLngBounds&, uint8_t z, uint16_t tileSize);
```

Comment:
Can we please use fixed width number types (like `uint64_t`)? Unfortunately, `unsigned long` is 32 bit on some and 64 bit on other platforms.